### ISSUE TITLE
build problem on xcode 12 beta 6

### ISSUE THREAD

**Issue by UtkuDalmaz** [Flinesoft/BartyCrouch#198]
I am having this on xcode 12 beta 6

```
dyld: Library not loaded: @rpath/lib_InternalSwiftSyntaxParser.dylib
  Referenced from: /usr/local/bin/bartycrouch
  Reason: image not found
/Users/utkudalmaz/Library/Developer/Xcode/DerivedData/APP-dfxahgqzuercupdhuslddgvvogqw/Build/Intermediates.noindex/APP.build/Debug-iphonesimulator/APP.build/Script-E53EEF331F83C8E5002B03E5.sh: line 7: 42759 Abort trap: 6           bartycrouch update -x
dyld: Library not loaded: @rpath/lib_InternalSwiftSyntaxParser.dylib
  Referenced from: /usr/local/bin/bartycrouch
  Reason: image not found
/Users/utkudalmaz/Library/Developer/Xcode/DerivedData/APP-dfxahgqzuercupdhuslddgvvogqw/Build/Intermediates.noindex/APP.build/Debug-iphonesimulator/APP.build/Script-E53EEF331F83C8E5002B03E5.sh: line 7: 42761 Abort trap: 6           bartycrouch lint -x
Command PhaseScriptExecution failed with a nonzero exit code
```

Any idea?

**Comment by lechuckcaptain** [comment]
Check if workaround proposed in #200 solve the issue also for you.

**Comment by Jeehut** [comment]
This should be fixed with version 4.3.0, please report otherwise if it still happens.
